src/appendices/git/workflows.rst

```diff
@@ -0,0 +1,170 @@
+.. _git-workflows:
+
+.. index::
+   single: Git; workflows
+
+Git Workflows
+=============
+
+This cheatsheet covers basic workflows for working with local and remote Git repositories.
```

Comment:
"workflows for working" sounds kinda funny.  How about, "...basic workflows for local and remote Git repositories."